data-science-onramp/data-cleaning/clean_test.py

```diff
@@ -2,28 +2,30 @@
 import re
 import uuid
 
+from google.cloud import bigquery
 from google.cloud import dataproc_v1 as dataproc
 from google.cloud import storage
+import pandas as pd
 import pytest
 
-# Set global variables
-PROJECT = os.environ["GCLOUD_PROJECT"]
-DATAPROC_CLUSTER = f"clean-test-{uuid.uuid4()}"
+# GCP project
+PROJECT_ID = os.environ["GCP_PROJECT"]
```

Comment:
nit - are these variable names the same as @Symmetries and Vismita used in Data Ingestion? If there's discrepancies, we should agree on unified var names across the tutorial

Reply:
They should all be standardized now